### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by mvcrouse**

### CODE REVIEW

**Review by gabe-l-hart — APPROVED** [review]
Looks merge ready to me!

**Review by drugilsberg — APPROVED** [review]
LGTM

**Review by drugilsberg — COMMENTED** [review]
See suggestion in the simple data builder README.md

**Review by hickeyma — APPROVED** [review]
LGTM, thanks for pushing @mvcrouse

File: fms_sdg/databuilders/simple/README.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+### Validators
+
+- `rouge_scorer`: validation of the generated data based on rouge.
+
+## Evaluation
+
+TBD
+
+# Notice
```
Comment (drugilsberg) [inline fms_sdg/databuilders/simple/README.md:41]:
```suggestion
## Notice
```

**Review by drugilsberg — COMMENTED** [review]
Added missing suggestion

**Pull request merged**